src/flags/coord_flag_env_handler.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+#pragma once
+
+namespace memgraph::flags {
+
+constexpr const char *kMgManagementPort = "MEMGRAPH_MANAGEMENT_PORT";
+constexpr const char *kMgCoordinatorPort = "MEMGRAPH_COORDINATOR_PORT";
+constexpr const char *kMgCoordinatorId = "MEMGRAPH_COORDINATOR_ID";
+
+struct CoordinationSetup {
+  int management_port;
+  int coordinator_port;
+  uint coordinator_id;
```

Comment:
I think ports should be uint16_t? Can you double check with the rest of the code?

Reply:
it is uint32_t so I changed it to that